src/components/Kurlsh.js

```diff
@@ -945,6 +945,10 @@ class Kurlsh extends React.Component {
             const currentOption = find(advancedOptions[addOn], (key, value) => value === data.flag);
             const doesCurrentErrExist = installerErrMsg ? installerErrMsg.includes(data.flag) : false;
 
+            if (option.hidden) {
+              return null;
+            }
+
```

Comment:
should this change be included? seems less related

Reply:
It hides the new option from the configuration screen. see https://github.com/replicatedhq/kurl.sh/pull/737/files#diff-edf7ecf74d0199ed4b45aea569c4030ab77a0a29ffb52af50d6c62d15f3fb885R548